python/setup.py

```diff
@@ -177,27 +199,31 @@ def cmake_configure(build_path, build_type, zlib_root, additional_compile_flags)
             set_additional_flags('extra_link_args', extra_link_arguments)
 
     if cmake_flags['KEYVI_LINK_LIBRARIES_DYNAMIC']:
-        set_additional_flags('libraries', cmake_flags['KEYVI_LINK_LIBRARIES_DYNAMIC'].split(' '))
+        set_additional_flags(
+            'libraries', cmake_flags['KEYVI_LINK_LIBRARIES_DYNAMIC'].split(' '))
 
     # set link args
     if cmake_flags['KEYVI_LINK_FLAGS']:
-        set_additional_flags('extra_link_args', cmake_flags['KEYVI_LINK_FLAGS'].split(' '))
+        set_additional_flags(
+            'extra_link_args', cmake_flags['KEYVI_LINK_FLAGS'].split(' '))
 
     return cmake_flags
 
 
 def set_additional_flags(key, additional_flags):
-     # patch the flags specified in key
+    # patch the flags specified in key
     for ext_m in ext_modules:
         flags = getattr(ext_m, key) + additional_flags
         setattr(ext_m, key, flags)
 
 
 def patch_for_custom_zlib(zlib_root):
     for ext_m in ext_modules:
-        include_dirs = [path.join(zlib_root, "include")] + getattr(ext_m, 'include_dirs')
+        include_dirs = [path.join(zlib_root, "include")] + \
+            getattr(ext_m, 'include_dirs')
```

Comment:
Honestly I don't like `\` in python code. 
So would ask why was this changed ? The line seems to be 90 chars long. 

also alternative can be to use `( ... + ... )` for long expressions or some temp vars.